site/content/docs/CICD/vercel.md

```diff
@@ -125,49 +125,58 @@ This is how a Checkly check maps to a Vercel check:
 
 ## Blocking Vercel deployments
 
-If you have the `Block my deployment when a check fails` checkbox enabled, Checkly informs Vercel whether a Vercel deployment should be "blocked" based on the following rules, per check type:
+You can block your Vercel deployments if any of your checks fail by enabling the checkboxes highlighted in the
+screenshot below. 
 
-### API checks
-For **API checks** any failing check results in a blocked Vercel deployment. This is a simple, binary state: it either fails or doesn't.
+![blocking deployments](/docs/images/cicd/vercel/vercel_block.png)
 
-### Browser checks
+We have two flavours here...
+
+1. **Block my deployment when a check fails**  does what it says on the tin and only apply to Vercel **Reliability checks**.
+2. **Block my deployment when Web Vitals degrade** is a bit special and only applies to Vercel **Performance checks** for our Browser checks. 
 
-For **Browser checks**, there are different ways a check can fail a Vercel deployment because the Reliability part and Performance part are evaluated separately.
+This table below show help you determine how blocking checks works. Note, again, that **Browser checks**, can fail a 
```

Comment:
typo: "show help you"

Reply:
typo: "Note, again, that browser checks, can fail a..."
There is an extra comma in this sentence.